contrib/packs/actions/pack_mgmt/download.py

```diff
@@ -44,10 +62,23 @@ def __init__(self, config=None):
         self._repo_url = None
 
     def run(self, packs, repo_url, abs_repo_base, verifyssl=True, branch='master', subtree=False):
+
+        cached_repo_url, cached_branch, cached_subtree = self._lookup_cached_gitinfo(
+            abs_repo_base, packs)
+
+        if not repo_url:
+            repo_url = cached_repo_url
+        if not branch:
```

Comment:
I still don't know how we hit this codepath with what you're proposing with an actionalias. As I see it, `branch` will always be satisfied, and this will never hit.